sdk/eventhub/azure-eventhubs/conftest.py

```diff
@@ -166,36 +165,36 @@ def device_id():
 @pytest.fixture()
 def connstr_receivers(connection_str):
     client = EventHubClient.from_connection_string(connection_str, debug=False)
-    eh_hub_info = client.get_eventhub_info()
+    eh_hub_info = client.get_eventhub_information()
     partitions = eh_hub_info["partition_ids"]
 
-    recv_offset = Offset("@latest")
+    recv_offset = EventPosition("@latest")
     receivers = []
     for p in partitions:
-        receivers.append(client.add_receiver("$default", p, prefetch=500, offset=Offset("@latest")))
-
-    client.run()
-
+        receiver = client.create_receiver("$default", p, prefetch=500, offset=EventPosition("@latest"))
+        receivers.append(receiver)
+        receiver.receive(timeout=1)
```

Comment:
If we are calling `receive` here, we don't need to do it again in like 178.

Reply:
Yes. Removed 178